src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
+    <TestsSuccessfulSemaphore Include="@(TestTargetFrameworkPath->'%(FullPath)\$(MSBuildProjectFile).tests.lastsucceeded')"/>
+    <XunitResultsFile Include="@(TestTargetFrameworkPath->'%(FullPath)\$(XunitResultsFileName)')"/>
+  </ItemGroup>
+
   <!-- On command line run unit tests on CoreCLR after the Test target -->
   <Target Name="RunTestsForProject"
           AfterTargets="CheckTestCategories"
+          Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath)"
```

Comment:
Can you please capture these inputs into an item group so they can be shared between this and the Copy target? Perhaps capture these in the TestCopyLocal item group. While you are at this could you handle https://github.com/dotnet/buildtools/issues/92 as well?